.github/workflows/integration.yaml

```diff
@@ -0,0 +1,61 @@
+name: integration
```

Comment:
worth adding a comment the purpose of splitting this pipeline into its own file

Reply:
like the one in line 15?